spyder/plugins/application/container.py

```diff
@@ -7,70 +7,153 @@
 """
 Container Widget.
 
-Holds references for base actions in the Main Menu of Spyder.
+Holds references for base actions in the Application of Spyder.
 """
 
+# Standard library imports
+import os
+
 # Third party imports
-from qtpy.QtCore import QThread, Slot
+from qtpy.QtCore import Qt, QThread, Slot
 from qtpy.QtWidgets import QMessageBox
 
 # Local imports
-from spyder import __project_url__
+from spyder import (
+    __docs_url__, __forum_url__, __project_url__, __trouble_url__)
 from spyder import dependencies
 from spyder.api.translations import get_translation
 from spyder.api.widgets import PluginMainContainer
 from spyder.config.base import DEV
 from spyder.config.utils import is_anaconda
+from spyder.utils.qthelpers import start_file, DialogManager
 from spyder.widgets.about import AboutDialog
 from spyder.widgets.dependencies import DependenciesDialog
 from spyder.widgets.helperwidgets import MessageCheckBox
 from spyder.workers.updates import WorkerUpdates
 
+WinUserEnvDialog = None
+if os.name == 'nt':
+    from spyder.utils.environ import WinUserEnvDialog
 
 # Localization
 _ = get_translation('spyder')
 
 
 # Actions
-class MainMenuActions:
+class ApplicationActions:
+    # Documentation related
```

Comment:
Please change this comment to `# Help` to be in the same line of the comments below (i.e. to what menu these actions are going to be added).